docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -323,4 +324,28 @@ realm-name:realm-two
 
 There are other options that can be specified for converting a filesystem realm, the details of which can be found using the `./bin/elytron-tool.sh filesystem-realm-integrity -h` command. 
 
+== Updating Elytron-tool Generated CLI Scripts for Domain Mode
+As mentioned above, some of the features added to `elytron-tool` create a CLI script that can be used to add a
```

Comment:
s/some of the features added to `elytron-tool`/some of the WildFly Elytron Tool commands

Reply:
"create a CLI script that can be used to add a key-store resource and a filesystem-realm resource in the Elytron subsystem that make use of the newly signed realm content"

I think we can be a bit more generic here, e.g., something like:

"create a CLI script that can be used to add resources to the Elytron subsystem"

Reply:
Fixed!